testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeSparkHive.java

```diff
@@ -82,7 +82,7 @@ public void extendEnvironment(Environment.Builder builder)
     private DockerContainer createSpark()
     {
         // TODO: Switch to pure Spark 3 image once it's available (https://github.com/trinodb/trino/issues/7063)
-        DockerContainer container = new DockerContainer("ghcr.io/trinodb/testing/spark3.0-iceberg:" + hadoopImagesVersion, "spark")
+        DockerContainer container = new DockerContainer("ghcr.io/trinodb/testing/spark3-iceberg:" + hadoopImagesVersion, "spark")
```

Comment:
It should require a new release, right? If so, it is blocked because the JDK in Pheonix images was updated to an incompatible version, see https://github.com/trinodb/trino/pull/13733

Reply:
I have a PR open there that fixes it: https://github.com/trinodb/docker-images/pull/143

Reply:
Yeah, you're right. Making this PR a draft until that release goes through